app/components/channel_list/header/__snapshots__/header.test.tsx.snap

```diff
@@ -1,134 +1,15 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`components/channel_list/header Channel List Header Component should match snapshot 1`] = `
-<View
-  animatedStyle={
-    Object {
-      "value": Object {
-        "marginLeft": 0,
-      },
-    }
-  }
-  collapsable={false}
+<RNCSafeAreaProvider
```

Comment:
Same for this test. Seems to be testing only the Safe Area provider, not the header.